src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/AggregationSingle.java

```diff
@@ -0,0 +1,63 @@
+/*-
+ * #%L
+ * OBKV Table Client Framework
+ * %%
+ * Copyright (C) 2021 OceanBase
+ * %%
+ * OBKV Table Client Framework is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation;
+
+import com.alipay.oceanbase.rpc.protocol.payload.AbstractPayload;
+import com.alipay.oceanbase.rpc.util.Serialization;
+
+import static com.alipay.oceanbase.rpc.util.Serialization.encodeObUniVersionHeader;
+import static com.alipay.oceanbase.rpc.util.Serialization.getObUniVersionHeaderLength;
+
+public class AggregationSingle extends AbstractPayload {
+    private AggregationType aggType_;
```

Comment:
I guess "-" is a C++ style. We do not use downline in JAVA, instead, Camel-Case is our choose